components/brave_ads/core/internal/ads/serving/choose/eligible_ads_predictor_util.h

```diff
@@ -133,13 +133,13 @@ double ComputePredictorScore(const AdPredictorInfo<T>& ad_predictor) {
   if (ad_predictor.ad_last_seen_hours_ago <= base::Time::kHoursPerDay) {
     score += ad_predictor_weights.at(kAdLastSeenHoursAgoIndex) *
              ad_predictor.ad_last_seen_hours_ago /
-             double{base::Time::kHoursPerDay};
+             static_cast<double>(base::Time::kHoursPerDay);
```

Comment:
`double{}` cast is safer and it is preferred over static_cast<double>

Reply:
Fixed, thanks